src/components/OrderForm/index.js

```diff
@@ -0,0 +1,53 @@
+import {useRouter} from 'next/router';
+import {useState} from 'react';
+import {useForm} from 'react-hook-form';
+
+import useStore from '../../hooks/useStore';
+import StyledButton from '../Button/styled';
+import CommentField from '../CommentField';
+import StyledOrderOptions from '../OrderOptions/styled';
+import SelectTable from '../SelectTable';
+import StyledSpan from '../StyledSpan/styled';
+
+import StyledOrderForm from './styled';
+
+export default function OrderForm() {
+	const [showFirstSpan, setShowFirstSpan] = useState(true);
+	function handleFirstSpan() {
+		setShowFirstSpan(!showFirstSpan);
+	}
+	const {register, watch} = useForm();
+	const selectHere = watch('selectHere');
+	const router = useRouter();
+	const clearCart = useStore(state => state.clearCart);
+	function checkoutMessage() {
+		alert('Thank you for your order. Your order-number is: 129');
```

Comment:
Maybe generate a serial number instead of giving always the same 129?

Reply:
Hey Petros! Thanks for the good Feedback. I have implemented it.